tests/plugins.test.js

```diff
@@ -0,0 +1,17 @@
+const supertest = require("supertest");
+const { configureSuperTest } = require("./helpers/supertest.wrapper");
+
+describe("Plugins tests", () => {
+  let request;
+  beforeEach(() => {
+    request = configureSuperTest(supertest);
+  });
+  describe("POST /api/plugins", () => {
+    it("should return list of plugins", async () => {
+      const { body, statusCode } = await request.post(`/plugins/`);
+      expect(statusCode).toBe(200);
+      // TODO: write a proper test with installing plugins
```

Comment:
Create an issue. If an issue is created, add the issue ID here.

Reply:
done https://kaholo.atlassian.net/browse/KAH-958

Reply:
Add the ID of the task to TODO (id is KAH-958).

Also, we write TODO as `@TODO:`